labzen/labzen.py

```diff
@@ -1,21 +1,75 @@
-import pandas as pd
+from nbformat import read, NO_CONVERT
+import os
+from pathlib import Path
+import re
+import glob
 
 
-def parse_lab(file_name: str) -> list:
+def parse_lab(notebook = 'no input'):
```

Comment:
I wonder if it might be better to change it to 

```suggestion
def parse_lab(notebook = None):
```

What do you think?